src/common/index.js

```diff
@@ -195,3 +194,17 @@ export function encodeFilename(name) {
 export function decodeFilename(filename) {
   return filename.replace(/-([0-9a-f]{2})/g, (_m, g) => String.fromCharCode(parseInt(g, 16)));
 }
+
+export async function getActiveTab() {
+  // not using destructuring as it adds 0.5kB of babel's _iterableToArrayLimit to injected*
```

Comment:
I think we should avoid such micro-optimizing and let the tools handle it.

Reply:
We should, in theory, but in reality the tools don't handle it, and I find it conceptually wrong that 99.99% of users who run Violentmonkey on modern browsers are forced to run the totally unnecessary polyfill for something that can be easily avoided.

Reply:
Ah, you mean we can switch to `Firefox>=53` in the other PR? That would be great! Let's handle that first then.